docs/guides/guides/module-api.mdx

```diff
@@ -56,30 +56,31 @@ node e2e-run-tests.js
+| `key`                     | _string_               | Specify your secret record key                                                                                                                                                                                               |
+| `exit`                    | _boolean_              | Whether to close Cypress after all tests run                                                                                                                                                                                 |
+| `parallel`                | _boolean_              | Run recorded specs in [parallel](/guides/cloud/smart-orchestration/parallelization) across multiple machines                                                                                                                 |
+| `port`                    | _number_               | Override default port                                                                                                                                                                                                        |
+| `project`                 | _string_               | Path to a specific project                                                                                                                                                                                                   |
+| `quiet`                   | _boolean_              | If passed, Cypress output will not be printed to `stdout`. Only output from the configured [Mocha reporter](/guides/tooling/reporters) will print.                                                                           |
+| `record`                  | _boolean_              | Whether to record the test run                                                                                                                                                                                               |
+| `reporter`                | _string_               | Specify a [Mocha reporter](/guides/tooling/reporters)                                                                                                                                                                        |
+| `reporterOptions`         | _object_               | Specify [Mocha reporter](/guides/tooling/reporters) options                                                                                                                                                                  |
+| `runnerUi`                | _boolean_              | Whether to display the Cypress Runner UI. Defaults to `true` when [Test Replay](/guides/cloud/debugging/test-replay) is not enabled. Defaults to `false` when [Test Replay](/guides/cloud/debugging/test-replay) is enabled. |
```

Comment:
More of a nit but this reads like the opposite first, almost like a double negative.
```suggestion
| `runnerUi`                | _boolean_              | Whether to display the Cypress Runner UI. Defaults to `false` when [Test Replay](/guides/cloud/debugging/test-replay) is enabled. Otherwise defaults to `true`. |
```